serverless/knative_eventing/serverless-using-brokers.adoc

```diff
@@ -1,16 +1,24 @@
 [id="serverless-using-brokers"]
-= Using brokers with Knative Eventing
+= Using brokers as an event delivery mechanism
 include::modules/common-attributes.adoc[]
 include::modules/serverless-document-attributes.adoc[]
 :context: serverless-using-brokers
 
 toc::[]
 
-Knative Eventing uses the `default` broker unless otherwise specified.
+You can use an xref:../../serverless/event_sources/knative-event-sources.adoc#knative-event-sources[event source] or a xref:../../serverless/knative_eventing/serverless-sinkbinding.adoc#serverless-sinkbinding[SinkBinding], the `default` broker, and a trigger together to decouple applications that produce events from the destination configuration.
 
-If you have cluster administrator permissions, you can create the `default` broker automatically using namespace annotation.
+Events are sent from an event source to the broker as an HTTP POST.
```

Comment:
You need a noun after "POST," maybe "request" or "method."

Reply:
@matzew or @cardil any recommendations here about which makes sense?

Reply:
Rquest in this sentence.